src/melee.cpp

```diff
@@ -2404,6 +2411,8 @@ void player_hit_message( Character *attacker, const std::string &message,
         if( attacker->is_npc() && !player_character.has_trait( trait_DEBUG_NIGHTVISION ) ) {
             //~ NPC hits something (critical)
             msg = string_format( _( "%s. Critical!" ), message );
+        } else if( technique && !wp_hit.empty() ) {
+            msg = string_format( _( "%s for %d damage to %s.  Critical!" ), message, dam, wp_hit );
```

Comment:
Please add translator comments.

Reply:
Thanks for the review! I read through `TRANSLATING.md`, but I'm not 100% confident on my usage of translation functions.

Reply:
I'd prefer something like "first %s is attacker, %d is amount of damage dealt, second %s is target's weak point".
Also could you please convert both of these %s into positional arguments?

Reply:
Done. I converted the new strings to positional, and formatted the comments as you suggested. I didn't change any of the existing strings.

Is there a style guide for translator comments / string formatting? Originally, I followed the format used in the surrounding code.

Reply:
> Done. I converted the new strings to positional, and formatted the comments as you suggested. I didn't change any of the existing strings.

Thanks. Much appreciated.

> Is there a style guide for translator comments / string formatting? Originally, I followed the format used in the surrounding code.

I'm afraid there isn't.